projects/Mallard/src/components/weather.tsx

```diff
@@ -125,7 +134,7 @@ const Weather = () => {
     }
 
     return weatherResponse({
-        error: ({ message }) => <></>,
+        error: ({}) => <></>,
```

Comment:
yep! not my code, just removed the unused dep for eslint. Error wise, since weather is not critical I wonder if failing this silently is almost good praxis? We can report this error to sentry/crashlytics before but UI-wise going blank might be better?

Reply:
Yes, this was by design by me. If weather fails, then simply hide it. But I agree once we add Sentry/Crashalytics we will need to log this.

Reply:
Do we have a ticket for Sentry/Crashalytics?

Reply:
We have this card in current sprint: https://trello.com/c/6hNmrILO